### PR TITLE
Remove "Cheatin' uh?" error message

### PR DESCRIPTION
**Pull request by ranss**
Minor changes to the code.

### CODE REVIEW

**Comment by arunbasillal** [comment]
@ranss I am curious. What was wrong with `Cheatin' uh?`?

**Comment by Tabrisrp** [comment]
Some Related WordPress ticket and changes in 4.9.5:
- https://core.trac.wordpress.org/ticket/38332
- https://core.trac.wordpress.org/changeset/42811

Pretty much any reference to this `Cheatin'` message has been removed from WP core now, and doesn't provide anything useful, so just exiting in this case is valid.

It doesn't have an impact outside of this, so I agree with the change.

File: wp-rocket.php [hunk 1]
```diff
@@ -29,7 +29,7 @@
 define( 'WP_ROCKET_WEB_VALID',             WP_ROCKET_WEB_MAIN . 'valid_key.php' );
 define( 'WP_ROCKET_WEB_INFO',              WP_ROCKET_WEB_MAIN . 'plugin_information.php' );
 define( 'WP_ROCKET_FILE',                  __FILE__ );
-define( 'WP_ROCKET_PATH',                  realpath( plugin_dir_path( WP_ROCKET_FILE ) ) . '/' );
+define( 'WP_ROCKET_PATH',                  realpath( dirname( WP_ROCKET_FILE ) ) . '/' );
```
Comment (Tabrisrp) [inline wp-rocket.php]: What is the idea behind changing this line?

Comment (arunbasillal) [reply wp-rocket.php]: Probably because `plugin_dir_path` already comes with a trailing slash?

Comment (hellofromtonya) [reply wp-rocket.php]: @arunbasillal True. It does:

```php
function plugin_dir_path( $file ) {
	return trailingslashit( dirname( $file ) );
}
```

Comment (hellofromtonya) [reply wp-rocket.php]: I'd prefer this type of change though to be in a separate PR, as it's not part of replacing `cheating, uh` work.

What do you think @Tabrisrp?

Comment (hellofromtonya) [reply wp-rocket.php]: @ranss I reverted commit 5c43c3dda5d3eec5586196d8a8a9f62a7368f2cd. Why? The code changes are not related to `cheatin' uh` replacement. Instead, those types of changes should be submitted as a separate PR.

**Comment by ranss** [comment]
> 
> 
> @ranss Thank you for contributing to WP Rocket! We appreciate your contribution +1

You are welcome @hellofromtonya 
Glad to constribute :)

**Comment by hellofromtonya** [comment]
@Tabrisrp A few notes for you when reviewing this PR:

- I reverted (removed) the code changes not related to swapping out the "cheatin' uh" code.
- I removed all "added" `defined( 'ABSPATH' ) || exit;` that are not currently in the codebase. Why? To minimize any unknown impacts.

**Pull request merged**